core/src/main/resources/defaults/blueprints/7.2.16/cdp-data-engineering-ha.bp

```diff
@@ -539,6 +550,7 @@
           "zeppelin-ZEPPELIN_SERVER-BASE",
           "das-DAS_EVENT_PROCESSOR",
           "das-DAS_WEBAPP",
+          "query_processor-QUERY_PROCESSOR",
```

Comment:
What's the additional functionality that the query-processor brings in here?
- Does it need to be an HA deployment. 
- What resource requirements does it have on the node?
- How is it different from the das-EVENTPROCESSOR ?

Reply:
- Initially the requirement was to add the Hue-QueryProcessor service in both cdp-data-engineering.bp and cdp-data-engineering-ha.bp for Datahub clusters wherever the DAS service presents. That's why we introduced it at both templates.
- It is a light-weight and optimised version to the DAS-eventprocessor. For now we kept it to get the customers familiar with both things, but in the upcoming release we will be sunsetting the entire DAS service.

**NOTE: This is not a new feature. This was introduced in 7.2.15 with complete testing and all the approvals. We just need it in 7.2.16 and all the upcoming release versions so that's why we are adding it in these two template files.**

Below is the original PR merged in 7.2.15 release:
https://github.com/hortonworks/cloudbreak/commit/33d2fcc78d47a55ceebc80552bf17bf171115500

Please check once with @lajosrodek for complete details and approve it.